File: iniciante/URI1048.java
package URI;

import java.util.Scanner;

public class URI1048 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double salario = scanner.nextDouble();
        double perc = 0;
        double reac;

        if (salario >= 0 && salario <= 400) {
            perc = 0.15;
        } else if (salario >= 400.01 && salario <= 800) {
            perc = 0.12;
        } else if (salario >= 800.01 && salario <= 1200) {
            perc = 0.10;
        } else if (salario >= 1200.01 && salario <= 2000) {
            perc = 0.07;
        } else if (salario > 2000) {
            perc = 0.04;
        }
        reac = salario * perc;
        salario += reac;

        System.out.printf("Novo salario: %.2f\n", salario);
        System.out.printf("Reajuste ganho: %.2f\n", reac);
        System.out.printf("Em percentual: %.0f %s\n", perc * 100, "%");
    }
}

File: python/Iniciante/URI1048.py
n = float(input())

if n >= 0 and n <= 400.0:
  reajuste = 0.15
elif n > 400.0 and n <= 800.0:
  reajuste = 0.12
elif n > 800.0 and n <= 1200.0:
  reajuste = 0.1
elif n > 1200.0 and n <= 2000.0:
  reajuste = 0.07
else:
  reajuste = 0.04

novo_salario = n + (n * reajuste)

print(f"Novo salario: {novo_salario:.2f}")
print(f"Reajuste ganho: {n * reajuste:.2f}")
print(f"Em percentual: {reajuste * 100:.0f} %")